components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -86,9 +86,13 @@ public String jwks() {
                     String errorMessage = "Invalid Tenant: " + tenantDomain;
```

Comment:
we can remove this since we are already doing the tenant validation in rewrite valve.. https://github.com/wso2-extensions/identity-carbon-auth-rest/blob/3d37c882f4b96701b8d333c2fc82b42b18da0348/components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/TenantContextRewriteValve.java#L111-L112